tryon_folder/test_pairs.txt

```diff
@@ -1 +1 @@
-person.png    shirt.png
+person_whole_front.png    kurti.jpg
```

Comment:
also just a note: this supports many test pairs, eg you could have
```text
person.png    shirt.png
person_whole_front.png    kurti.jpg
```
and (if we adjusted the notebook) all of the scripts should be able to process all pairs at the same time, which could be faster maybe

Reply:
Yup! The scripts in the notebook can process all the images which are listed, since there were only 2 images I restricted it, but the idea is that the instructions under `Add a new image` can keep updating the `test_pairs`. I'll run a sample of multiple images once we have some nicer results too